src/main/scala/com/campudus/tableaux/database/domain/folder.scala

```diff
@@ -24,15 +25,10 @@ case class Folder(id: FolderId,
   )
 }
 
-case class ExtendedFolder(folder: Folder,
-                          subfolders: Seq[Folder],
-                          files: Seq[ExtendedFile]) extends DomainObject {
+case class ExtendedFolder(folder: Folder, subfolders: Seq[Folder], files: Seq[ExtendedFile]) extends DomainObject {
   override def getJson: JsonObject = {
     val folderJson = folder.getJson
 
-    folderJson.mergeIn(Json.obj(
-      "subfolders" -> compatibilityGet(subfolders),
-      "files" -> compatibilityGet(files)
-    ))
+    folderJson.mergeIn(Json.obj("subfolders" -> compatibilityGet(subfolders), "files" -> compatibilityGet(files)))
```

Comment:
It was more readable the way it was before. Is it possible to not have it put everything in one line here?